smalltalksrc/VMMaker/VMStackPage.class.st

```diff
@@ -34,6 +34,17 @@ VMStackPage class >> alignedByteSizeOf: anObject forClient: aVMClass [
 	^self surrogateClass alignedByteSize
 ]
 
+{ #category : #translation }
+VMStackPage class >> filteredInstVarNames [
+
+	| instVarNames |
+	instVarNames := super filteredInstVarNames.
+
+	^ BytesPerWord = 4
+		  ifTrue: [ ^ instVarNames copyWithout: #padToWord ]
+		  ifFalse: [ ^ instVarNames ]
```

Comment:
The returns are necessary outside or inside, but not in both.

```suggestion
	^ BytesPerWord = 4
		  ifTrue: [ instVarNames copyWithout: #padToWord ]
		  ifFalse: [ instVarNames ]
```

Maybe the Formater can check this 🤔